packages/react-aria-components/package.json

```diff
@@ -39,8 +39,10 @@
   "dependencies": {
     "@internationalized/date": "^3.5.5",
     "@internationalized/string": "^3.2.3",
+    "@react-aria/accordion": "3.0.0-alpha.33",
```

Comment:
can we remove this dep now? Noticed it in `@react-spectrum/accordion`'s package json too